pkg/yurttunnel/trafficforward/dns/util_test.go

```diff
@@ -231,26 +231,26 @@ func TestAddOrUpdateRecord(t *testing.T) {
 		record  string
 		expect  []string
 	}{
-		//{
-		//	desc: "test add record",
-		//	records: []string{"10.1.218.68\tk8s-xing-61"},
-		//	record: "10.1.10.62\tk8s-xing-62",
-		//	expect: []string{"10.1.218.68\tk8s-xing-61","10.1.10.62\tk8s-xing-62"},
-		//},
-		//
-		//{
-		//	desc: "test update record",
-		//	records: []string{"10.1.218.68\tk8s-xing-61"},
-		//	record: "10.1.10.62\tk8s-xing-61",
-		//	expect: []string{"10.1.10.62\tk8s-xing-61"},
-		//},
+		{
+			desc:    "test add record",
+			records: []string{"10.1.218.68\tk8s-xing-61"},
+			record:  "10.1.10.62\tk8s-xing-62",
+			expect:  []string{"10.1.218.68\tk8s-xing-61", "10.1.10.62\tk8s-xing-62"},
+		},
 
 		{
-			desc:    "test idempotence",
+			desc:    "test update record",
 			records: []string{"10.1.218.68\tk8s-xing-61"},
 			record:  "10.1.10.62\tk8s-xing-61",
-			expect:  []string{"10.1.218.68\tk8s-xing-61"},
+			expect:  []string{"10.1.10.62\tk8s-xing-61"},
 		},
+
+		//{
+		//	desc:    "test idempotence",
+		//	records: []string{"10.1.218.68\tk8s-xing-61"},
+		//	record:  "10.1.10.62\tk8s-xing-61",
+		//	expect:  []string{"10.1.218.68\tk8s-xing-61"},
+		//},
```

Comment:
I don't know what is the purpose of this test case "test idempotence" 😕 . Maybe we can just delete the case.

Reply:
Ok